src/screens/dashboard/CreateButton.tsx

```diff
@@ -2,21 +2,25 @@ import React from 'react';
 import { FloatingAction } from 'react-native-floating-action';
 
 import { Icon } from 'components';
-import { Student } from 'models';
 import { palette } from 'styles';
 
-export class CreateStudentButton extends React.PureComponent {
+interface Props {
+  onPress: () => void;
+  actionName: string;
+}
+
+export class CreateButton extends React.PureComponent<Props> {
```

Comment:
It is so simple right now, that you can go with function component. It will be cleaner.